osmnx/core.py

```diff
@@ -221,6 +224,11 @@ def osm_net_download(polygon=None, north=None, south=None, east=None, west=None,
     else:
         maxsize = '[maxsize:{}]'.format(memory)
 
+    # use custom settings if delivered, otherwise just the default ones.
+    overpass_settings = '[out:json][timeout:{timeout}]{maxsize}'.format(timeout=timeout, maxsize=maxsize)
```

Comment:
this should probably be converted to an if/else pattern rather than an if/overwrite pattern

Reply:
Done.